pkg/api/licenses.go

```diff
@@ -884,3 +884,61 @@ func SearchInLicense(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, res)
 }
+
+// ExportLicenses gives users all licenses as a json file.
+//
+//	@Summary		Export all licenses as a json file
+//	@Description	Export all licenses as a json file
+//	@Id				ExportLicenses
+//	@Tags			Licenses
+//	@Accept			json
+//	@Produce		json
+//	@Success		200	{array}		models.LicenseJson
+//	@Failure		500	{object}	models.LicenseError	"Failed to fetch obligations"
+//	@Security		ApiKeyAuth
```

Comment:
```suggestion
//	@Failure		500	{object}	models.LicenseError	"Failed to fetch licenses"
```
It should be licenses and the endpoint in unauthorized.